src/discordcommandhandler.ts

```diff
@@ -125,9 +125,8 @@ export class DiscordCommandHandler {
             }));
             let errorMsg = "";
             await Promise.all(allChannelMxids.map(async (chanMxid) => {
-                const intent = this.bridge.getIntent();
                 try {
-                    await intent[funcKey](chanMxid, name);
+                    await this.bridge.botIntent.underlyingClient[funcKey + "User"](chanMxid, name);
```

Comment:
did you test if the `funcKey`s need updating?

Reply:
Good catch!

Reply:
Yep, all happy